lib/docs/admin/api.md

```diff
@@ -0,0 +1,364 @@
+
```

Comment:
Looks like you had a bad rebase for the docs directory. All the changes in here should already exist on develop in different file names.

Reply:
Rebased off of latest and removed docs